### ISSUE TITLE
Installation Guide: Add page on possible integrations to set up after installing Dataverse

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#4283]
As someone who just installed Dataverse (or asked someone to install it for me), I'd like to know what integrations are available to me now that I have a new shiny Dataverse toy to play with.

The list of possible integrations should be a dedicated page in the Installation Guide with a mention leading up to the "going live" step at http://guides.dataverse.org/en/4.8.2/installation/config.html#going-live-launching-your-production-deployment

Integrations can also be set up **after** you go life, by why not launch with an integration or two already set up for your users? 😄 

I started a thread on the Google Group to make sure I haven't forgotten any integrations: https://groups.google.com/d/msg/dataverse-community/tDb8WKWFnqw/ySOeBNeIAQAJ

Here's the list (feel free to update this if you have access or leave a comment below):

- Google Analytics and/or Piwik
- TwoRavens/Zelig
- WorldMap
- Open Journal Systems (OJS)
- Open Science Framework (OSF)
- RSpace
- SHARE
- "compute" (:ComputeBaseUrl setting)

Dataverse can be used with other tools such as Zotero but I think that would be better mentioned in the User Guide. The list above is stuff that sysadmins need to install or configure or make arrangements with third parties (OSF and SHARE, for example) to get working.

Each item above should include a blurb about what it is with a link to where to find more information on how to get it working with a new installation of Dataverse.

**Comment by pdurbin** [comment]
> Open Science Framework (OSF)

I just noticed that both links under http://guides.dataverse.org/en/4.8.6/api/apps.html#osf are 404s:

<img width="678" alt="screen shot 2018-05-25 at 8 27 58 am" src="https://user-images.githubusercontent.com/21006/40544231-8f451d06-5ff5-11e8-9e20-92383f6e4b38.png">

The good news, however, is that it's now possible to add any arbitrary Dataverse installation to OSF! I just commented on https://github.com/CenterForOpenScience/osf.io/issues/2636#issuecomment-392040617 with the following screenshot and I'm wondering how @shlake from https://dataverse.lib.virginia.edu and my team at Harvard feels about those two installations being listed while the other 31 installations of Dataverse are not:

<img width="1163" alt="screen shot 2018-05-25 at 8 16 09 am" src="https://user-images.githubusercontent.com/21006/40544304-cf97db5a-5ff5-11e8-990b-490a852ba2b8.png">

This issue is on my mind because my "Integrations for Dataverse and Empowering the Community" talk was accepted and is now listed at https://projects.iq.harvard.edu/dcm2018/agenda . I'd like to be able to link to a new page in the guides explaining these integrations in greater depth. While I'm in there, I'll move content around and make sure there are no 404s. If anyone wants to give me feedback or ideas for my talk, you can see the abstract and the slides I'm working on at this Google doc: https://docs.google.com/document/d/1hFS16uyDGNW2OJmfsjTqM1Ck_KdnXkuGKggmqiLkVI0/edit?usp=sharing

A couple weeks ago, I gave a variation of this talk, starting at slide 67 at https://www.slideshare.net/philipdurbin/managing-sharing-and-curating-your-research-data-in-a-digital-environment

**Comment by shlake** [comment]
I'm Traveling now on vacation. Will comment in detail when I get back to the office next week.

**Comment by ross-spencer** [comment]
@pdurbin as per #4715. We've some work to do here, but we started looking at creating Archival AIPs from a Dataverse integration back in 2017. Most of the information about it is on the Archivematica wiki: https://wiki.archivematica.org/Dataverse and we'll figure out how to share information in time. @joel-simpson is a great contact for this type of thing and has a keen interest in research data from Archivematica's side.

**Comment by pdurbin** [comment]
I just made and merged pull request #4737 ( @matthew-a-dunlap looked it over for me, thanks!) to stub out a page on integrations in the Admin Guide. I'd love to hear feedback.

@shlake I'm going to close this issue but you're welcome to comment here or on the integrations thread at https://groups.google.com/d/msg/dataverse-community/tDb8WKWFnqw/ySOeBNeIAQAJ

@ross-spencer it's absolutely awesome that you're working away on the Archivematic integration but for now I listed it on the "dev efforts from the community" spreadsheet linked from the "Future Integrations" section in the pull request. I hope you don't mind that I made you the primary contact. Here's a direct link to the spreadsheet: https://docs.google.com/spreadsheets/d/1pl9U0_CtWQ3oz6ZllvSHeyB0EG1M_vZEC_aZ7hREnhE/edit?usp=sharing
